rpc/client/event_test.go

```diff
@@ -51,7 +51,7 @@ func testTxEventsSent(ctx context.Context, t *testing.T, broadcastMethod string,
 	}()
 
 	// and wait for confirmation
-	evt, err := client.WaitForOneEvent(c, types.EventTxValue, waitForEventTimeout)
+	evt, err := client.WaitForOneEvent(ctx, c, types.EventTxValue, waitForEventTimeout)
```

Comment:
mlrf. I'm gonna have to merge this into my omnium gatherum change 😢

Reply:
Nah, it's fine, I just made more changes to this function than just the context.